packages/design-system/tsconfig.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": "tsconfig/react-library.json",
+  "compilerOptions": {
+    "lib": ["dom", "es2017"]
```

Comment:
we need "dom" as a lib as well, the default one didn't include it and using browser APIs was not working